build.gradle

```diff
@@ -26,6 +26,7 @@ repositories {
 ext {
     camelVersion = '3.18.1'
     zeebClientVersion = '8.1.1'
+    springBootVersion = '2.6.2'
```

Comment:
Spring boot version in this  service is id 'org.springframework.boot' version '2.7.3'
see line number 10

Reply:
Updated